packages/react-components/react-utilities/etc/react-utilities.api.md

```diff
@@ -138,7 +138,7 @@ export type Slots<S extends SlotPropsRecord> = {
 };
 
 // @public
-export type SlotShorthandValue = React_2.ReactChild | React_2.ReactNodeArray | React_2.ReactPortal;
+export type SlotShorthandValue = React_2.ReactChild | React_2.ReactNode[] | React_2.ReactPortal;
```

Comment:
`ReactNodeArray` is deprecated and throwing an error, replaced with `ReactNode[]	`

**Error:** 
`@deprecated — Use either ReactNode[] if you need an array or Iterable<ReactNode> if its passed to a host component.`